bentoml/_internal/cli/bento_management.py

```diff
@@ -98,6 +99,11 @@ def list_bentos(bento_name, output, no_trunc):
                 bentos, key=lambda x: x.info.creation_time, reverse=True
             )
         ]
+
+        if not no_trunc:
```

Comment:
Why is the `no_trunc` option not a flag? `bentoml list --no-trunc` seems more natural to me than `bentoml list --no-trunc true`

Reply:
@sauyon I think it should be a flag